src/Particular.CodeRules/Particular.ruleset

```diff
@@ -1,6 +1,212 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Particular Software Rule Set" Description=" CodeAnalysis Rules for Particular Software Projects" ToolsVersion="14.0">
-  <Include Path="minimumrecommendedrules.ruleset" Action="Default" />
```

Comment:
This is marked as "legacy" in the FxCop package, so I think we should just remove it.

Reply:
@adamralph I see your comment on the Include which you already removed.